Engine/src/Engine/Physics/Ray.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "glm/glm.hpp"
+
+namespace Engine
+{

```

Comment:
What's ray used for? Is it just for the mouse click location?

Reply:
The one class I forgot to comment. If its still vague please ping.